<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExistingRooms</class>
 <widget class="QDialog" name="ExistingRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Your username:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leUsername"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Choose a game you want to join:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listOfRooms">
       <property name="styleSheet">
        <string notr="true">color: rgb(46, 52, 54);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pbJoin">
     <property name="text">
      <string>Join Game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
